gamelogic.js

```diff
@@ -1,9 +1,15 @@
+// Functies om de ingevoerde letters van het te raden woord weer te geven
```

Comment:
Korte, heldere comments waardoor er snel overzicht is en de bedoeling van de code duidelijk is.